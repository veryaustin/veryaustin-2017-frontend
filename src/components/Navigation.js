import React, {Component} from 'react';
import {IndexLink} from 'react-router';
import MenuButton from './MenuButton';

class Navigation extends Component {
 render() {
   return(
     <div>
     <MenuButton />
      <nav id="main_nav">
        <IndexLink to="/">Home</IndexLink>
        <a href="#">Work</a>
        <a href="#">About</a>
        <a href="http://writing.veryaustin.com">Writing</a>
        <a href="#">Contact</a>
      </nav>
    </div>
   );
 }
}

export default Navigation;