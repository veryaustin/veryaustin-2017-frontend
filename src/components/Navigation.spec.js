import React from 'react';
import {shallow} from 'enzyme';
import Navigation from './Navigation';

describe('Navigation Component', () => {
  it('should render the navigation component', () => {
    const wrapper = shallow(<Navigation />);
    const navigation = wrapper.type();
    const expected = 'nav';
    expect(navigation).toEqual(expected);
  });

});