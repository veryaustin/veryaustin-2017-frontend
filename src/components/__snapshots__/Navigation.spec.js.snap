exports[`Navigation Component renders the navigation component 1`] = `
<div>
  <MenuButton
    onClick={[Function]} />
  <styled.nav
    navToggled={false}>
    <Styled(IndexLink)
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      onClick={[Function]}
      to="/">
      Home
    </Styled(IndexLink)>
    <Styled(Link)
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      onClick={[Function]}
      to="work">
      Work
    </Styled(Link)>
    <Styled(Link)
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      onClick={[Function]}
      to="about">
      About
    </Styled(Link)>
    <Styled(Link)
      className=""
      href="http://writing.veryaustin.com">
      Writing
    </Styled(Link)>
    <Styled(Link)
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      onClick={[Function]}
      to="contact">
      Contact
    </Styled(Link)>
  </styled.nav>
</div>
`;
